Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -50,9 +50,7 @@ import SwiftSyntaxBuilder
 ///
 /// ### After
 /// ```swift
-/// { someInt in
-///   <#T##String#>
-/// }
+/// <#{ <#someInt#> in <#T##String#> }#>
```

Comment:
If we expand the editor placeholder, we shouldn’t end up with an editor placeholder again.

Reply:
Oops, reverted!